app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,21 @@
+from .dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.hummer_level = hummer_level

```

Comment:
Make sure that you store arguments in the protected attributes as mentioned in the checklist

Reply:
not resolved